uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab-filters.component.js

```diff
@@ -7,21 +7,23 @@ define([
 ], function (_, moment, components) {
   components.component('leaveBalanceTabFilters', {
     controller: LeaveBalanceTabFiltersController,
-    controllerAs: 'leaveBalanceTabFilters',
+    controllerAs: 'filtersVm',
```

Comment:
@reneolivo We had this conversation before with @AkA84 (was something like `actions.actions` and decided it is fine)